sklearn/cross_validation.py

```diff
@@ -332,6 +332,119 @@ def __len__(self):
         return self.n_folds
 
 
+def disjoint_label_folds(y, n_folds=3):
```

Comment:
+1, you cannot predict something you have never seen on the training set. `y` in scikit-learn is always the target variable for supervised prediction. I agree, it's a bad name but it's too late to change.